docs/newContributors.md

```diff
@@ -0,0 +1,48 @@
+<h1 align="center">Hello Peeps 👋</h1>
+<h2 align="center">Welcome to the <a href="https://github.com/siv2r/kidney-exchange">kidney-exchange</a> Repo 🎉</h2>
+
+<h3 align="center">Here is, Complete Guide to Contribute to this Repo.</h3>
+
+<br>
+
+- ##### Firstly, Join our [Discord](https://discord.com/invite/d7j63WZV82) Server, Introduce yourself there, Interact with Others as 
+    > **Open Source is more about Collaboration.**
+
+- ##### So to start Contibuting to our Project you can Solve existing issues or if you have found any bug or  want to add new featurre to Project you can create New Issue , that's purely your choice.
+  ![](../images/Issues.png)
+
+- ##### Project Maintainers and Mentors Will Guide you With the issue and Incase of any problem you face ,you can take help from the community in the discord server.
+  
+- ##### So, Now You have an Issue to work upon. So First you need to fork Repo. "*Fork means producing a Personal Copy of Someone else's Project*". You can fork Repo This Way.
+  ![](../images/Fork.png)
+
+- ##### Clone the Repo you have forked :-
+     - Copy the link of the forked Repo.
+     - Go to the desired location in the terminal, where you want to place project.
+     - Run command ``git clone link_of_forked_repo``.
+     - Run Command ``cd kidney-exchnage ``.
+     - Open the folder wth your favourite text editor command.
```

Comment:
The heading is smaller than the points, could you make the heading bigger in the whole document